engine/src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -7,6 +7,7 @@ import akka.actor.ActorSystem
 import better.files._
```

Comment:
I renamed this actor in my PR, could you take a look if that works for you?... I had to rename it due to the initialization actor.